KsApi/mutations/inputs/CreatePaymentIntentInput.swift

```diff
@@ -4,21 +4,25 @@ public struct CreatePaymentIntentInput: GraphMutationInput, Encodable {
   let projectId: String
   let amountDollars: String
   let digitalMarketingAttributed: Bool?
+  let paymentIntentContext: GraphAPI.StripeIntentContextTypes?
 
   /**
    Initializes a CreateCheckout.
 
    - parameter projectId: The GraphID of the Project.
    - parameter amountDollars: The amount.
    - parameter digitalMarketingAttributed: The optional ID of the ShippingRule's Location.
+   - parameter paymentIntentContext: The optional GraphAPI.StripeIntentContextType. Used to help the backend debug
    */
   public init(
     projectId: String,
     amountDollars: String,
-    digitalMarketingAttributed: Bool?
+    digitalMarketingAttributed: Bool?,
+    paymentIntentContext: GraphAPI.StripeIntentContextTypes?
   ) {
     self.projectId = projectId
     self.amountDollars = amountDollars
     self.digitalMarketingAttributed = digitalMarketingAttributed
+    self.paymentIntentContext = paymentIntentContext
```

Comment:
Is this ever _not_ `.postCampaignCheckout`? i.e. do we ever use a payment intent for a live pledge? You could potentially just hard-code it here.

Reply:
That's a good point! I doubt we'll need a payment intent in a different context unless, for some weird reason, we want to create one from the account settings -> payment methods screen.